### PR TITLE
Added notes for creating accessible packages

### PR DESCRIPTION
**Pull request by RachBreeze**
Hello

This documentation was originally written for the Umbraco Packages team but following a review with @LottePitcher, @KevinJump , @dawoe and so on that the pull request should be moved to Umbraco docs.

I am fully expecting feedback on the PR. The goal wasn't to replace what is available on the internet and from others but to provide some basic information on testing and the benefits of using the Umbraco UI.

### CODE REVIEW

**Comment by eshanrnh** [comment]
Thanks for the PR, @RachBreeze. We will review this soon 💪

**Comment by sofietoft** [comment]
Hi @RachBreeze !

Thanks again for the PR!
I've made some minor suggestions - but might have gotten a little carried away and made too many.
Please take a look and tell me what you think!

Finally, in the final bullet you mention something called "Umbraco Style" - is that something specific you're referring to here?
It's not something I've heard mentioned like that before, so just wanted to check 💪

**Comment by sofietoft** [comment]
Hi @RachBreeze !
Did you see my message?

Would love to get this article merged, but I need some feedback from your side 🙌

**Comment by RachBreeze** [comment]
Hi @sofietoft 

So sorry for the slow reply. I've been battling post Christmas bugs and COVID, but these amends look great (I know you've accepted the PR already, but felt I still needed to let you know :-) )

**Comment by sofietoft** [comment]
No worries @RachBreeze ! 😄 
Thanks for taking a look!

I've talked to Ronald about sending it by the packages community team as well - this is kinda their domain, so I just want to make sure we're publishing something they can also account for 💪

File: umbraco-cms/extending/packages/accessibility.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+## Testing
+
+Accessibility testing is more a specialist skillset than it is automated testing. The purpose of this document is to outline what can be done to help build accessible packages. It is not a complete list of accessibility tests that can be performed.
+
+- Build the components using the [Umbraco UI components](https://uui.umbraco.com/) as these have accessibility tests built within them.
+- Use the keyboard to tab through the elements on the page checking:
+  - Does the element tabbed to have a **focus state**?
+  - Does the **tab order** make sense?
+- Check the UI with a screen reader.
+  - [NVDA have a free screen reader](https://www.nvaccess.org/download/) and some guidelines on screen reader testing are available from [webaim](https://webaim.org/articles/screenreader_testing/)
```
Comment (ronaldbarendse) [inline umbraco-cms/extending/packages/accessibility.md:18]: Lets add some acronym definitions:
```suggestion
  - [NVDA: NonVisual Desktop Access is a free Windows screen reader](https://www.nvaccess.org/download/) and some guidelines on screen reader testing are available from [WebAIM: Web Accessibility In Mind](https://webaim.org/articles/screenreader_testing/).
```

File: umbraco-cms/extending/packages/accessibility.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+- Install use an accessibility testing tool as a plugin into your browser to run automated tests.
```
Comment (ronaldbarendse) [inline umbraco-cms/extending/packages/accessibility.md:19]:
```suggestion
- Install an accessibility testing tool as a plugin into your browser to run automated tests:
```

File: umbraco-cms/extending/packages/accessibility.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  - Tools like [axe DevTools](https://chrome.google.com/webstore/detail/axe-devtools-web-accessib/lhdoppojpmngadmnindnejefpokejbdd) are built to reduce the number of false positives in a test.
+- If the UI does not follow the Umbraco Style, then check the contrast with a tool like [WCAG Contrast Checker](https://chrome.google.com/webstore/detail/wcag-color-contrast-check/plnahcmalebffmaghcpcmpaciebdhgdf) will help ensure contrast.
```
Comment (ronaldbarendse) [inline umbraco-cms/extending/packages/accessibility.md:21]:
```suggestion
- If the UI does not follow the Umbraco default colors/styles, test the color contrast between foreground and background according to the Web Content Accessibility Guidelines (WCAG) using a tool like [WCAG Color contrast checker](https://chrome.google.com/webstore/detail/wcag-color-contrast-check/plnahcmalebffmaghcpcmpaciebdhgdf).
```

**Review by ronaldbarendse — COMMENTED** [review]
It's great to have this included in the documentation and I can see this getting expanded with some examples later (probably when the new back-office is in place and using the Umbraco UI library is the default way to create new UI) 💪🏻

I've only added some suggestions to fix the acronym warnings, the rest looks good to me!

File: umbraco-cms/extending/packages/accessibility.md (hunk 1 above)
Comment (RachBreeze) [reply umbraco-cms/extending/packages/accessibility.md:18]: I hadn't thought about the acronym's thank you

File: umbraco-cms/extending/packages/accessibility.md (hunk 3 above)
Comment (RachBreeze) [reply umbraco-cms/extending/packages/accessibility.md:21]: Ah now you've bought in the debate on colour v color...  ;-) 

This is so much better though thank you

**Comment by RachBreeze** [comment]
Hi @sofietoft  and @ronaldbarendse  thank you both for your review. I realised that what focus state and tab order are was undefined, so I've updated for more information

**Comment by RachBreeze** [comment]
And failed to pull your comments in first! Argh

File: umbraco-cms/extending/packages/accessibility.md (hunk 1 above)
Comment (RachBreeze) [reply umbraco-cms/extending/packages/accessibility.md]: @ronaldbarendse how would we say WCAG is Web Content Accessibility Guidelines (the last bullet point) following this style for the acronyms.

Comment (sofietoft) [reply umbraco-cms/extending/packages/accessibility.md]: I've solved the acronym warnings 💪

**Comment by sofietoft** [comment]
I've fixed the warnings about the undefined acronyms, and also added the article to the SUMMARY file in order to actually add it to the docs site!

I'll merge this one in, as soon as the final checks are complete 🙌 
Thanks again for adding this to the docs @RachBreeze !

**Pull request merged**